loginLegacy/src/main/java/com/brandon/loginlegacy/view/LoginScreenActivity.kt

```diff
@@ -25,64 +27,72 @@ class LoginScreenActivity : AppCompatActivity() {
         binding = LoginScreenBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
-        val emailField = binding.loginEmailField
-        val emailError = binding.loginEmailError
-        val codeField = binding.loginCodeField
-        val fieldsError = binding.loginRequiredFieldsError
-        val buttonSpinner = binding.progressSpinner
-        val buttonText = binding.buttonText
-        val loginButton = binding.loginButton
+        loginViewModel.viewModelScope.launch {
+            loginViewModel.uiState.collect {
+                updateUi(it)
+                Log.e(LoginViewModel.VIEWMODEL_ERROR_TAG, "${loginViewModel.uiState}")
```

Comment:
Is it really an error to receive a state? lol

Reply:
Removed the word error lol